src/components/ConversationSettings/ListableSettings.vue

```diff
@@ -20,25 +20,27 @@
 -->
 
 <template>
-	<div>
-		<div>
-			<NcCheckboxRadioSwitch :checked="listable !== LISTABLE.NONE"
-				:disabled="isListableLoading"
-				type="switch"
-				@update:checked="toggleListableUsers">
-				{{ t('spreed', 'Open conversation to registered users, showing it in search results') }}
-			</NcCheckboxRadioSwitch>
-		</div>
-		<div v-if="listable !== LISTABLE.NONE">
-			<div v-if="isGuestsAccountsEnabled">
-				<NcCheckboxRadioSwitch :checked="listable === LISTABLE.ALL"
-					:disabled="isListableLoading"
-					type="switch"
-					@update:checked="toggleListableGuests">
-					{{ t('spreed', 'Also open to guest app users') }}
-				</NcCheckboxRadioSwitch>
-			</div>
-		</div>
+	<div v-if="canFullModerate">
+		<NcCheckboxRadioSwitch :checked="listable !== LISTABLE.NONE"
+			:disabled="isListableLoading"
+			type="switch"
+			@update:checked="toggleListableUsers">
+			{{ t('spreed', 'Open conversation to registered users, showing it in search results') }}
+		</NcCheckboxRadioSwitch>
+		<NcCheckboxRadioSwitch v-if="listable !== LISTABLE.NONE && isGuestsAccountsEnabled"
+			class="additional-top-margin"
+			:checked="listable === LISTABLE.ALL"
+			:disabled="isListableLoading"
+			type="switch"
+			@update:checked="toggleListableGuests">
+			{{ t('spreed', 'Also open to guest app users') }}
+		</NcCheckboxRadioSwitch>
+	</div>
+	<div v-else>
+		<h5 class="app-settings-section__subtitle">
+			{{ t('spreed', 'Conversation visibility scope') }}
```

Comment:
Let's not use a 3rd word for this please. We already have "Listable conversations" (API) and "open conversations". Should stick to "open conversations"

Reply:
"Conversation open scope" then? Don't know, how to correctly put this in subheading

Reply:
> ### Open conversation
> 
> * Conversation is open to registered and guest app users
> * Conversation is open to registered users
> * Conversation is not open for registered users to join

Reply:
Renamed the heading

Closed conversation is already aligned with toast message, I'd leave it like this:

`showSuccess(t('spreed', 'You limited the conversation to the current participants'))`
https://github.com/nextcloud/spreed/blob/master/src/components/ConversationSettings/ListableSettings.vue#L138